storage/connect/frids.h

```diff
@@ -1,30 +1,30 @@
-    case IDS_TABLES:  p = "Table Ent�tes";    break;
```

Comment:
Same as above. Let's add a comment referencing the intended interpretation/encoding of these strings.

Reply:
The character-set-fixing has been split out into https://github.com/MariaDB/server/pull/2188, so it can be resolved.